lib/elastomer/version_support.rb

```diff
@@ -151,9 +165,18 @@ def unsupported_indexing_directives
       @unsupported_indexing_directives = unsupported_keys.each_with_object({}) do |key, h|
         h[key] = "_#{key}"
       end
+      check_op_type(@unsupported_indexing_directives) # hack: valid param loses underscore between ES 2.x & 5.x
       @unsupported_indexing_directives.freeze
     end
 
+    # COMPATIBILITY
+    # If _op_type param is found in map of symbols to request param names, check for ES 5.x and shim
+    def check_op_type(params = {})
```

Comment:
I would maybe call this `fix_op_type` or something that indicates it mutates the input. Some Rubyists would also add a `!` on the end of the method to indicate this, not sure if we do here.

Reply:
ah good call thank you!